### DISCUSSION TITLE
Stripe FDW

### DISCUSSION THREAD

**Discussion by magick93** [supabase/supabase#12712] [General]
Hello

I'm running `supabase 1.38.7` and have setup the stripe wrapper by following the instructions on https://supabase.github.io/wrappers/stripe/

Yet when I run `select * from stripe.customers limit 10;` (or any other simply query) I get the following error:

```
request failed: Request error: error sending request for url (https://api.stripe.com/v1/invoices?limit=100): error trying to connect: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed:../ssl/statem/statem_clnt.c:1914: (unable to get local issuer certificate)
```

Can anyone help with this?

**Comment by burmecia** [comment]
That looks like SSL certificate issue, would you able to upgrade your project to latest version and try again?

**Comment by encima** [comment]
Closing due to age, please re-open if this is still occurring
